ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nutanix-4.18-nightly-x86.yaml

```diff
@@ -70,7 +73,7 @@ tests:
     test:
     - chain: openshift-qe-cluster-density-v2
     workflow: openshift-qe-installer-nutanix
-  timeout: 6h0m0s
+  timeout: 8h0m0s
```

Comment:
lets set these timeouts back to what they were since we were focusing on node-density-heavy test here